README.rst

```diff
@@ -52,7 +52,7 @@ Requirements
 
 Shapely 1.7 requires
```

Comment:
Shouldn't it be "Shapely 1.8 requires ..." then?

Reply:
Ah, yes, indeed ;) 
Thanks @sgillies for the fixup